crates/api_common/src/community.rs

```diff
@@ -19,7 +18,6 @@ pub struct GetCommunity {
   pub id: Option<CommunityId>,
   /// Example: star_trek , or star_trek@xyz.tld
   pub name: Option<String>,
-  pub auth: Option<Sensitive<String>>,
```

Comment:
I'm sure I'll figure this out further down, but I don't know how you're going to check that some routes require auth, and some don't now.